src/webpack.config.js

```diff
@@ -113,6 +124,6 @@ module.exports = {
   optimization: {
     minimize: isProduction,
     minimizer: [terserPlugin],
-    noEmitOnErrors: true,
+    emitOnErrors: true,
   },
```

Comment:
In my testing, I had to add `usedExports: false` to disable some aggressive tree shaking. Did you run into the same thing at all?

Reply:
In the `webpack v5` migration, nothing was written about `usedExports`. If I'm not mistaken, it was also included by default for productions in version 4 (https://v4.webpack.js.org/configuration/optimization/#optimizationusedexports), so I'm not sure if we should turn it off.